src/Rebus.Datadog.Tracing/Rebus.Datadog.Tracing.csproj

```diff
@@ -11,8 +11,8 @@
     </PropertyGroup>
 
     <ItemGroup>
-        <PackageReference Include="Datadog.Trace" Version="2.5.1" />
-        <PackageReference Include="Rebus" Version="6.6.2" />
+        <PackageReference Include="Datadog.Trace" Version="2.24.1" />
```

Comment:
This probably shouldn't be upgraded unless there are breaking changes

Reply:
But why leave it on an old version?

Reply:
Mainly because ideally, it matches the version installed in the consuming application

Reply:
Let's go with that then 😄